client/src/components/QuestionsAnswers.jsx

```diff
@@ -3,12 +3,10 @@ import axios from 'axios';
 import './Sub_Questions/styles/questions.css';
 import QuestionsSearch from './Sub_Questions/QuestionSearch.jsx';
 import QuestionList from './Sub_Questions/QuestionList.jsx';
-import AskQuestion from './Sub_Questions/AskQuestion.jsx';
 
 export default function QuestionsAnswers({ productID, productName }) {
   const [questionList, setQuestionList] = useState([]);
   const [filteredList, setFilteredList] = useState([]);
-  const [show, setShow] = useState(false);
 
   const updateQuestions = () => {
     axios.get(`/api/qa/questions?product_id=${productID}`)
```

Comment:
I believe a "fetchQuestions" is more appropriate than an "updateQuestions" since it's a get requset.

Reply:
will update that :)